test/e2e_node/hugepages_test.go

```diff
@@ -192,7 +192,12 @@ func getHugepagesTestPod(f *framework.Framework, limits v1.ResourceList, mounts
 var _ = SIGDescribe("HugePages [Serial] [Feature:HugePages][NodeSpecialFeature:HugePages]", func() {
 	f := framework.NewDefaultFramework("hugepages-test")
 
-	ginkgo.It("should remove resources for huge page sizes no longer supported", func() {
+	/*
+	   Release : v1.22
+	   Testname: Removal of hugepages resources
+	   Description: Kubelet should remove hugepages resources of deallocated hugepages.
+	*/
+	framework.ConformanceIt("should remove resources for huge page sizes no longer supported [NodeConformance]", func() {
```

Comment:
Is this a [LinuxOnly] feature?

Reply:
Thank you for the review.
It is. Updated the PR.